File: PracticeOne/p1-java/src/main/java/practica/tdd/Calculator.java
package practica.tdd;

public class Calculator {

    public int add(int a, int b) {
        return a + b;
    }

    public int divide(int dividend, int divisor) {
        if (divisor == 0) {
            throw new ArithmeticException("/ by zero");
        }
        return dividend / divisor;
    }
}

File: PracticeOne/p1-kotlin/src/main/kotlin/practica/tdd/Calculator.kt
package practica.tdd

class Calculator {
    fun add(a: Int, b: Int): Int = a + b

    fun divide(dividend: Int, divisor: Int): Int {
        if (divisor == 0) {
            throw ArithmeticException("/ by zero")
        }
        return dividend / divisor
    }
}
